jbpm-workitem-itests/pom.xml

```diff
@@ -212,7 +211,9 @@
           <artifactId>maven-surefire-plugin</artifactId>
           <configuration>
             <systemPropertyVariables>
+              <toxiproxy.image>shopify/toxiproxy:2.1.0</toxiproxy.image>
               <toxiproxy.port>9093</toxiproxy.port>
+              <org.kie.executor.jms>false</org.kie.executor.jms>
```

Comment:
just curious - why disabling jms support for the executor?

Reply:
if not, this warning is shown: 
`WARN  Disabling JMS support in executor because: unable to initialize JMS configuration for executor due to Need to specify class name in environment or system property, or as an applet parameter, or in an application resource file:  java.naming.factory.initial`